packages/definitions-parser/src/check-parse-results.ts

```diff
@@ -161,7 +161,7 @@ async function checkNpm(
   log("  yarn not-needed " + yarnargs.join(" "));
   log(`  git add --all && git commit -m "${name}: Provides its own types" && git push -u origin not-needed-${name}`);
   log(`  And comment PR: This will deprecate \`@types/${name}\` in favor of just \`${name}\`. CC ${contributorUrls}`);
-  if (semver.gt(`${major}.${minor}.0`, firstTypedVersion)) {
+  if (semver.gte(`${major}.${minor}.0`, firstTypedVersion)) {
```

Comment:
Make the condition slightly more correct, same as https://github.com/microsoft/DefinitelyTyped-tools/blob/575fd10bc80d3fb50135a429855a3b0864983a95/packages/definitions-parser/src/git.ts#L153 and https://github.com/microsoft/DefinitelyTyped-tools/blob/575fd10bc80d3fb50135a429855a3b0864983a95/packages/publisher/src/lib/npm.ts#L15.